src/algebra/ordered_group.lean

```diff
@@ -1808,7 +1801,32 @@ abs_sub_le_iff.2 ⟨sub_le_sub hau hbl, sub_le_sub hbu hal⟩
 lemma eq_of_abs_sub_nonpos (h : abs (a - b) ≤ 0) : a = b :=
 eq_of_abs_sub_eq_zero (le_antisymm h (abs_nonneg (a - b)))
 
-end decidable_linear_ordered_add_comm_group
+lemma exists_gt_zero [nontrivial α] : ∃ (a:α), 0 < a :=
+begin
+  obtain ⟨y, hy⟩ := exists_ne (0 : α),
+  cases hy.lt_or_lt,
+  { exact ⟨- y, neg_pos.mpr h⟩ },
+  { exact ⟨y, h⟩ }
+end
+
+/- TODO This theorem could be written in the context of monoids rather than groups; however,
+`linear_ordered_add_comm_monoid` does not currently exist in mathlib. -/
```

Comment:
```suggestion
```
I copied and adapted this comment from the previous docstring, but maybe it's not true!